docs/bottom-tab-navigator.md

```diff
@@ -12,7 +12,8 @@ createBottomTabNavigator(RouteConfigs, BottomTabNavigatorConfig);
 
 ## RouteConfigs
 
-The route configs object is a mapping from route name to a route config.
+
```

Comment:
can you please use a relative link? (just`stack-navigator.html#routeconfigs`)

Reply:
Alright, done!